scripts/upgrade

```diff
@@ -63,11 +63,7 @@ python3 -m venv $install_dir/venv
 #=================================================
 ynh_script_progression --message="Updating a configuration file..."
 
-mkdir -p "/etc/$app"
-ynh_add_config --template="../conf/headphones.ini" --destination="/etc/$app/headphones.ini"
-
-chmod 600 "/etc/$app/headphones.ini"
-chown $app:$app "/etc/$app/headphones.ini"
+ynh_replace_string --match_string="^http_root = .*" --replace_string="http_root = $path" --target_file="/etc/$app/headphones.ini"
```

Comment:
(side note but the hash of the config file is not going to be updated, ie set as manually modified ... however as titus seems to point out, this file is edited by the app itself (or the admin via the app idk whatever) so it probably doesn't really matter ...)